ParisBNF/abb/BNFabb106.xml

```diff
@@ -0,0 +1,226 @@
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                            <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the front board.
+                                </desc>
+                                </item>
+                                <item xml:id="e2">
+                                    <desc type="StampExlibris">
+                                        Shelfmark of the <placeName ref="INS0303BNF">Library</placeName> glued onto the inner side of the front board.
+                                    </desc>
+                                </item>
+                            </list>
+                            </additions>
+                        <bindingDesc>
+                            <binding  contemporary="true">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote>  
+                                <decoNote xml:id="b2" type="Cover">Leather quarter cover.</decoNote>  
+                                <decoNote xml:id="b3" type="SewingStations" cert="medium">4</decoNote>  
+                                <decoNote xml:id="b4" cert="medium">Two pairs of sewing stations.</decoNote>  
+                                <decoNote xml:id="b5" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>  
+                                <decoNote xml:id="b6" type="SlipCase">According to <persName
+                                    ref="PRS1071dAbbadi">A. d’Abbadie</persName>,
+                                    the manuscript has a new traditional two-part leather case.</decoNote>              
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
+                    <history>
+                        <origin>
+                            <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was commissioned by  <persName
+                                ref="PRS1071dAbbadi">A. d'Abbadie</persName> in Ethiopia. According to him, the text of the manuscript was copied for him ʻto correct the strange omission that had been made of this section in manuscript no. 87.ʼ<foreign xml:lang="fr">(ʻréparer l'étrange oubli qu'on avait fait de cette section dans le manuscrit n. 87ʼ)</foreign>.
+                        </origin>
+                        <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
```

Comment:
ok for the library here, you can also consider enhancing the date by `<date when=1902">`